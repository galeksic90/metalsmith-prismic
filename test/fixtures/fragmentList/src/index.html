---
template: index.hbt
prismic:
  store:
    query: '[[:d = any(document.type, ["store"])]]'
    arrayFragments: true
---
