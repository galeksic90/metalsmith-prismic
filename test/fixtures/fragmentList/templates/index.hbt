<html>
<body>
<ul>
{{#each prismic.store.results}}
    <li>
        {{{data.name.html}}}
        <ul>
          <li>{{{data.monday.0.html}}}{{{data.monday.1.html}}}</li>
          <li>{{{data.tuesday.0.html}}}{{{data.tuesday.1.html}}}</li>
          <li>{{{data.wednesday.0.html}}}{{{data.wednesday.1.html}}}</li>
          <li>{{{data.thursday.0.html}}}{{{data.thursday.1.html}}}</li>
          <li>{{{data.friday.0.html}}}{{{data.friday.1.html}}}</li>
        </ul>
    </li>
{{/each}}
</ul>
</body>
</html>
