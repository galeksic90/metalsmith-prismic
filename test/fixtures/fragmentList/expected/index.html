<html>
<body>
<ul>
    <li>
        <h1>Paris Saint-Lazare</h1>
        <ul>
          <li><span>2pm - 10pm</span></li>
          <li><span>2pm - 10pm</span></li>
          <li><span>2pm - 10pm</span></li>
          <li><span>2pm - 10pm</span></li>
          <li><span>2pm - 10pm</span></li>
        </ul>
    </li>
    <li>
        <h1>Tokyo Roppongi Hills</h1>
        <ul>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
        </ul>
    </li>
    <li>
        <h1>London Covent Garden</h1>
        <ul>
          <li><span>9am - 8pm</span></li>
          <li><span>9am - 8pm</span></li>
          <li><span>9am - 8pm</span></li>
          <li><span>9am - 8pm</span></li>
          <li><span>9am - 8pm</span></li>
        </ul>
    </li>
    <li>
        <h1>Paris Champ-Elys√©es</h1>
        <ul>
          <li><span>9am - 2pm</span><span>3pm - 7pm</span></li>
          <li><span>9am - 2pm</span><span>3pm - 7pm</span></li>
          <li><span>9am - 2pm</span><span>3pm - 7pm</span></li>
          <li><span>9am - 2pm</span><span>3pm - 7pm</span></li>
          <li><span>9am - 2pm</span><span>3pm - 7pm</span></li>
        </ul>
    </li>
    <li>
        <h1>New York Fifth Avenue</h1>
        <ul>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
          <li><span>9am - 10pm</span></li>
        </ul>
    </li>
</ul>
</body>
</html>
